src/adapters/voyage/helpers/embedding.ts

```diff
@@ -0,0 +1,25 @@
+import { VoyageAIClient } from "voyageai";
+import { Context } from "../../../types";
+import { SuperVoyage } from "./voyage";
+const VECTOR_SIZE = 1024;
+
+export class Embedding extends SuperVoyage {
+  protected context: Context;
+
+  constructor(client: VoyageAIClient, context: Context) {
+    super(client, context);
+    this.context = context;
+  }
+
+  async createEmbedding(text: string | null): Promise<number[]> {
+    if (text === null) {
+      return new Array(VECTOR_SIZE).fill(0);
```

Comment:
Still confused about this fill stuff

Reply:
If a text is empty, instead of calling embedding api on it I am using fill to create zeros array of 1024-dim. We have to do this as embedding has to be a not null value for vector search to work. And zero vectors do not cause any change to vector search result as well.